lib/project_types/node/commands/create.rb

```diff
@@ -67,12 +67,18 @@ def check_node
         version, stat = @ctx.capture2e('node', '-v')
         @ctx.done("node #{version}")
         @ctx.abort(NODE_REQUIRED_NOTICE) unless stat.success?
+      rescue Errno::ENOENT
```

Comment:
I think this is a good way to do it could mask other errors. What we could do instead so that we can make better error messages for the user is `@ctx.system('which', 'node').success?` This will allow us to specifically check if the tool exists.

Reply:
I think that would fail in Windows, do we to support that?

Reply:
Yes, the original error was in Windows. I also thought of that @tanema, but we are actually printing out the version, and this enables us to do it with a single command. I do agree that 'which' would be the ideal choice but I'm not aware of a check that works across different OSs.

Reply:
Our Windows support is currently through Windows Subsystem for Linux (WSL), so it's not native Windows but Linux.  So the `which` check would work.

Reply:
Excellent point! I think we should then go ahead and do as @tanema proposes.